corehq/apps/users/views/mobile/users.py

```diff
@@ -1184,6 +1184,9 @@ class FilteredUserDownload(BaseManageCommCareUserView):
     @method_decorator(require_can_edit_commcare_users)
     def get(self, request, domain, *args, **kwargs):
         form = CommCareUserFilterForm(request.GET, domain=domain, couch_user=request.couch_user)
+        # On First load of the page we were getting an error No role is specified.
+        # Setting empty_permitted to true takes case of it.
```

Comment:
nit: care.

also a rephrase
`To avoid errors on first page load`
I think should be enough